### PR TITLE
chore(botocore): Format botocore

### PR DESCRIPTION
**Pull request by P403n1x87**
## Description

Format botocore

## Checklist
- [ ] ~~Added to the correct milestone.~~
- [ ] ~~Tests provided or description of manual testing performed is included in the code or PR.~~
- [ ] ~~Library documentation is updated.~~
- [ ] ~~[Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).~~

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,23 +1,20 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "setuptools_scm[toml] >= 4", "cython"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 40.6.0", "setuptools_scm[toml] >= 4", "cython"]
```
Comment (Kyle-Verhoog) [inline pyproject.toml:3]: looks like these were sorted, which is fine by me

**Review by Kyle-Verhoog — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (P403n1x87) [reply pyproject.toml:3]: yep, looks like one of the VS Code extensions is formatting this file on save

Comment (jd) [reply pyproject.toml:3]: honestly, if it'd be just a line it'd be fine but if every developer with different settings keeps changing files around it's going to be a mess…
Can we avoid that?
I hate to be the grumpy guy sorry 😢

**Review by jd — CHANGES_REQUESTED** [review]

**Comment by P403n1x87** [comment]
So what should I do in this case? Revert the sorting or open a PR just for the formatting of `pyproject.toml`?

File: pyproject.toml (hunk 1 above)
Comment (brettlangdon) [reply pyproject.toml:3]: Unless we enforce linting/formatting on toml files to ensure everyone is consistent, let's revert the sorting change.

(is there a vscode settings file we can add to the repo to disable that, or... something to ensure a consistent experience for vscode users?)

Comment (P403n1x87) [reply pyproject.toml]: I have just added
~~~ json
    "[toml]": {
        "editor.formatOnSave": false
    },
~~~
to my vscode settings

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

**Review by jd — APPROVED** [review]

**Pull request merged**